umbraco/Umbraco-CMS
# v10: Nullable reference types - Umbraco.Infrastructure

# Notes
- Enabled nullable in the Infrastructure project
- Some of the problems i ran into this time is time is that in the DataValueEditorFactory we have the Create method that takes an array as a param, i wanted to make this a nullable array but this breaks cause we cannot find a valid constructor then.
The reason i want a nullable array is that we use this on all of our value editors with the Attribute property, which is nullable.
![image](https://user-images.githubusercontent.com/70372949/155994043-00cfc614-4397-4bb3-b5d5-1cfd7098f450.png)
For now all of these has been supressed with `!`
# How to test
- Install some of the popular umbraco packages like uSync and try using them, ensure they still work
